CountlyFeedbackWidget.m

```diff
@@ -144,25 +140,25 @@ - (void)recordResult:(NSDictionary * __nullable)result
 - (NSURLRequest *)dataRequest
 {
     NSString* queryString = [NSString stringWithFormat:@"%@=%@&%@=%@&%@=%@&%@=%@&%@=%@&%@=%@",
-        kCountlyQSKeySDKName, CountlyCommon.sharedInstance.SDKName,
-        kCountlyQSKeySDKVersion, CountlyCommon.sharedInstance.SDKVersion,
-        kCountlyFBKeyAppVersion, CountlyDeviceInfo.appVersion,
-        kCountlyFBKeyPlatform, CountlyDeviceInfo.osName,
-        kCountlyFBKeyShown, @"1",
-        kCountlyFBKeyWidgetID, self.ID];
+                             kCountlyQSKeySDKName, CountlyCommon.sharedInstance.SDKName,
```

Comment:
are those things related to formatter rigjt?

Reply:
yes, most of the changes are related to auto formatting